src/ws/cockpitchannelresponse.c

```diff
@@ -125,6 +129,21 @@ cockpit_channel_inject_perform (CockpitChannelInject *inject,
   cockpit_web_response_add_filter (response, filter);
   g_object_unref (filter);
   g_free (prefixed_application);
+
+  // Lang hack
+  if (inject->lang)
```

Comment:
Problem with this is that this only works in shell. I have not been able to figure out where and how .html pages for iframes are being handled and server so I could edit their content as well.

@mvollmer do you have some idea?